.tekton/roxctl-pull-request.yaml

```diff
@@ -0,0 +1,398 @@
+apiVersion: tekton.dev/v1beta1
+kind: PipelineRun
+metadata:
+  annotations:
+    build.appstudio.openshift.io/repo: https://github.com/stackrox/stackrox?rev={{revision}}
+    build.appstudio.redhat.com/commit_sha: '{{revision}}'
+    build.appstudio.redhat.com/pull_request_number: '{{pull_request_number}}'
+    build.appstudio.redhat.com/target_branch: '{{target_branch}}'
+    pipelinesascode.tekton.dev/max-keep-runs: "3"
+    pipelinesascode.tekton.dev/on-event: '[pull_request]'
+    pipelinesascode.tekton.dev/on-target-branch: '[*]'
+  creationTimestamp: null
+  labels:
+    appstudio.openshift.io/application: acs
+    appstudio.openshift.io/component: roxctl
+    pipelines.appstudio.openshift.io/type: build
+  name: roxctl-on-pull-request
+  namespace: rh-acs-tenant
+spec:
+  params:
+  - name: dockerfile
+    value: image/roxctl/rhtap.Dockerfile
+  - name: git-url
+    value: '{{repo_url}}'
+  - name: image-expires-after
+    value: 13w
+  - name: output-image
+    value: quay.io/redhat-user-workloads/rh-acs-tenant/acs/roxctl:on-pr-{{revision}}
+  - name: path-context
+    value: .
+  - name: revision
+    value: '{{revision}}'
+  # TODO(ROX-20234): Enable hermetic builds
+  # - name: hermetic
+  #   value: "true"
+  # - name: prefetch-input
+  #   value: '{"type": "gomod", "path": "."}'
```

Comment:
@tommartensen could you please remind why is prefetch disabled here?

Reply:
In a previous version, `prefetch-input` was only executed when `hermetic` was set to `true`. 
The documentation says it should be possible to enable only `prefetching`: https://redhat-appstudio.github.io/docs.appstudio.io/Documentation/main/how-to-guides/proc_prefetching-dependencies-to-support-hermetic-build/#enabling-prefetch-builds-for-gomod.

I will check this.